packages/elements-dev-portal/src/styles.css

```diff
@@ -1 +1,12 @@
 @import "../../elements-core/src/styles.css";
+
+.sl-elements-article {
+    justify-content: space-evenly;
```

Comment:
Idk. I find it a bit weird.

First of all, we should keep CSS to bare minimum and use it only if mosaic's styling solution really can't solve our problems.

On top of that, this will only affect dev-portal. Meanwhile I think the biggest importance was for that to look good in platform. But it doesn't matter - for me the layout should be the same in platform, dev-portal and elements.

Reply:
Ok, so there are few things here that need addressing:

> First of all, we should keep CSS to bare minimum and use it only if mosaic's styling solution really can't solve our problems.

Most definitely. Mosaic applies to MDV though, and I feel that enforcing a specific width or padding there isn't a perfect solution. It would reduce the flexibility of customization of MDV outside of Elements. Unless it's directly required by Design or Product person, I think we should aim for Elements styling to be as customizable as possible.

> On top of that, this will only affect dev-portal. Meanwhile I think the biggest importance was for that to look good in platform. But it doesn't matter - for me the layout should be the same in platform, dev-portal and elements.

The thing is that Platform uses `NodeContent`. If we change this for `dev-portal` it will be present in Platform after Elements bump. Even if I'd want to style this directly within Platfrom, I will still need specific selectors for these containers.

One thing this PR will do though is that it will enforce styling for `NodeContent` - ideally we probably want to have a custom styling within `dev-portal` and `Platform` but not for `NodeContent` directly - so that people can still import that component and style it as they want